packages/configs/package.json

```diff
@@ -17,6 +17,7 @@
         "eslint": "^7.32.0",
         "eslint-config-airbnb": "^18.2.1",
         "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-config-prettier": "^6.12.0",
```

Comment:
Not related, but it was indeed missing here and blocking me from proceeding, so decided to fix it at once.